src/api/services/adminServiceClient.ts

```diff
@@ -83,10 +108,43 @@ export const addKeyPair = async (
   name: string,
   disabled: boolean = false
 ): Promise<KeyPairDTO> => {
-  const response = await adminServiceClient.post<KeyPairDTO>('/api/client_side_keypairs/add', {
-    site_id: siteId,
-    disabled,
-    name,
-  });
-  return response.data;
+  try {
+    const response = await adminServiceClient.post<KeyPairDTO>('/api/client_side_keypairs/add', {
+      site_id: siteId,
+      disabled,
+      name,
+    });
+    return response.data;
+  } catch (error: unknown) {
+    const [logger] = getLoggers();
```

Comment:
sorry - you'll now have some merge conflicts. You'll probably want to change this to `const { errorLogger } = getLoggers();`

Reply:
I made the update.  Thanks for the heads-up.